### ISSUE TITLE
BUILD FAILED

### ISSUE THREAD

**Issue by msink** [JetBrains/kotlin-native#1883]
```
> Task :performance:compileKonanRingMingw_x64
performance\src\main\kotlin-native\cleanup.kt:3:8: error: unresolved reference: konan
import konan.internal.GC                                                                                    12:25
       ^
performance\src\main\kotlin-native\cleanup.kt:5:17: error: unresolved reference: GC
fun cleanup() { GC.collect() }
                ^
performance\src\main\kotlin\org\jetbrains\ring\Utils.kt:24:6: error: unresolved reference: konan
    @konan.ThreadLocal
     ^
performance\src\main\kotlin\org\jetbrains\ring\Utils.kt:36:6: error: unresolved reference: konan
    @konan.ThreadLocal
     ^

> Task :performance:compileKonanRingMingw_x64 FAILED
```

**Comment by sbogolepov** [comment]
Thanks, it's remainings of #1845.

**Comment by olonho** [comment]
Fixed with 4a461026582d1adb1e7adafe096e56e905d99289
